tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1104,3 +1104,9 @@ Feature: check share activity
       | permissionsRole | Viewer       |
     When user "Brian" tries to list the activities of file "anotherTextfile.txt" from space "Personal" owned by user "Alice" using the Graph API
     Then the HTTP status code should be "403"
+
+  @issue-9860
+  Scenario: user tries to check activities of another user's file
```

Comment:
this is not a sharing scenario. should go to another feature file.

Reply:
Oh yeah!  Thanks.
I will move it to `apiActivities/activities.feature`